day_7/high_level.md

```diff
@@ -0,0 +1,83 @@
+## Caesar Cipher
+
+### My thoughts on the one of the oldest methods of data encryption
+A Caesar Cipher is going to take our input and create a coded message.  In order to do this I looked up a few different methods.  Below is a method I found from this website. `https://medium.com/@alexander.virga/ruby-simple-string-encryption-shift-caesar-cipher-encoder-rot-9dedf06374d1`  I have a fairly basic understanding of this so I thought I would copy the method I copied from the website and explain what each line is doing. What I have found is that being able to type out the code and explain it that way.
+
```

Comment:
Again, great job being transparent about working through a found solution. 

This exercise is to get you to think about algorithmic thinking - like writing down a recipe - writing each step you need to accomplish to make the cipher. You should then be able to take your steps (written in plain language), and write code for each step. 

Good work writing what each piece of code does along with example outputs. A challenge would be to write the "what each step does" part first, and _then_ come up with the code.